src/components/Pages/Admin/Designer.vue

```diff
@@ -532,34 +543,34 @@ export default {
                 if (error.code === 400) {
                     this.$log.debug('error code 400 ', error.message);
 
+                    await this.updateWorkshop([groupId, { xml: xmlNoConstr, properties }]);
                     await this.removeThenAddElements(groupId);
 
-                    // await this.updateWorkshop([groupId, { xml }]);
-
                     correct = 1;
                 } else {
                     this.showToast('Error while creating/updating workshop', 'warning');
                     console.log(error.message);
                 }
-
                 return correct;
             }
         },
 
+        getElementIdToSendIt(groupId, label) {
```

Comment:
I think it's enough just to say `getElementId`. Don't presuppose where this function will be used, that's up to the client.